common/lib/xmodule/xmodule/assetstore/assetmgr.py

```diff
@@ -49,14 +49,8 @@ class AssetManager(object):
     @contract(asset_key='AssetKey', throw_on_not_found='bool', as_stream='bool')
     def find(asset_key, throw_on_not_found=True, as_stream=False):
         """
-        Finds a course asset either in the assetstore -or- in the deprecated contentstore.
+        gets a course asset in the deprecated contentstore.
+        Note - Changed due to multiple calls to unpickling compressed course structure
```

Comment:
I was hoping for something more high level here:
This method used to do x + y, now it just does y. We can't have it do x anymore because z. To work around z in the future, we may have to refactor this functionality to do w.